marshmallow_enum/__init__.py

```diff
@@ -88,7 +91,7 @@ def _deserialize_by_value(self, value, attr, data):
             self.fail('by_value', input=value, value=value)
 
     def _deserialize_by_name(self, value, attr, data):
-        if not isinstance(value, str_types):
+        if not isinstance(value, six.string_types):
```

Comment:
Looking at six, I don't think there's an actual difference but what is declared in this module and what six uses. Unless someone decided to subclass basestring themselves in Python2, in which case six would catch it and this package would not.

If you insist on this change, I'd prefer changing the string_types for Python2 to just basestring rather than dragging six into this.